pyramid_orb/api.py

```diff
@@ -150,22 +150,22 @@ def collect_documentation(self, name, service_info):
                 yield group_name, section
 
     def process(self, request):
-        # return the schemas for this API
+        # return the schema information for this API
         if not request.traversed and \
             request.method.lower() == 'get' and \
             'application/json' in request.accept and \
             request.params.get('returning') == 'schema':
-            schemas = [s.__json__() for s in sorted(orb.system.schemas().values(), key=lambda x: x.name())]
+            schemas = [s.__json__() for s in sorted(orb.system.schemas().values(), key=lambda x: x.name()) if hasattr(s.model(), '__resource__')]
```

Comment:
so, this is a new thought that will break things -- let me know what you think.

Instead of determining if a model should be exposed to the frontend & rest API, I will look at __resource__ vs. __auth__.  __auth__ is still used to drive authentication to editing, but not exposure:

```
class User(orb.Table):
    __resource__ = True
```

vs.

```
class User(orb.Table):
    __auth__ = None
```

It seemed to make more sense to separate those two out.